rails_programming/introduction_to_rails/project_feet_wet.md

```diff
@@ -22,51 +22,54 @@ In this chapter, you'll build a simple scaffolded app on Rails.  It won't be as
 </div>
 
 ### Student Solutions
-Send us your solution by editing this [file](https://github.com/TheOdinProject/curriculum/blob/master/rails_programming/introduction_to_rails/project_feet_wet.md) so we can show others! Submit a link to the Github repo with your files in it here using any of the methods listed on the [contributing page](http://github.com/TheOdinProject/curriculum/blob/master/contributing.md).
+Send us your solution by editing this [file](https://github.com/TheOdinProject/curriculum/blob/master/rails_programming/introduction_to_rails/project_feet_wet.md) so we can show others! Submit a link to the Github repository with your files in it here using any of the methods listed on the [contributing page](http://github.com/TheOdinProject/curriculum/blob/master/contributing.md).
 
-If you will be submitting a solution for the RoR tutorial chapter 2, make sure it's in a repo that you won't overwrite when we move onto chapter 3 (which starts a new app)
+Submit your solution for the Ruby on Rails tutorial chapter 2, make sure it is in a repository that you will not overwrite when we move onto chapter 3 (which starts a new app).
```

Comment:
Agree with the full stop and RoR. But I would keep the contractions as it's standard English and commonly used. I think it's hard to read when you remove the contractions.

Reply:
As a brazilian guy, It's nice to know about this standard English.